srcpkgs/xsnow/template

```diff
@@ -1,23 +1,14 @@
 # Template file for 'xsnow'
 pkgname=xsnow
-version=1.42
-revision=3
-hostmakedepends="imake xorg-cf-files"
-makedepends="libXext-devel libXpm-devel libXt-devel"
+version=3.4.2
+revision=1
+archs="i686 x86_64*"
```

Comment:
Why are arches limited?

Reply:
Builds fail on other archs.

Reply:
Set nocross=yes instead.

Reply:
```diff
diff --git a/srcpkgs/xsnow/template b/srcpkgs/xsnow/template
index 83ed8f9237..4a60cb0a50 100644
--- a/srcpkgs/xsnow/template
+++ b/srcpkgs/xsnow/template
@@ -2,8 +2,8 @@
 pkgname=xsnow
 version=3.4.2
 revision=1
-archs="i686 x86_64*"
 build_style=gnu-configure
+build_helper=qemu
 hostmakedepends="pkg-config"
 makedepends="libXext-devel libXpm-devel libXt-devel libX11-devel gtk+3-devel"
 short_desc="Snow for your X11 sessions"
@@ -12,3 +12,9 @@ license="GPL-3.0-or-later"
 homepage="https://ratrabbit.nl/ratrabbit/xsnow/index.html"
 distfiles="https://ratrabbit.nl/downloads/xsnow/${pkgname}-${version}.tar.gz"
 checksum=c618d7a73cef7070755f38ddc3665487afa8c71a9c82ae9055723e594a62fd9e
+
+post_patch() {
+       if [ "$CROSS_BUILD" ]; then
+               vsed -e "s:./xsnow -manpage:/usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static &:" -i src/Makefile.am src/Makefile.in
+       fi
+}
```

Reply:
> Set nocross=yes instead.

This will make it not available on musl arch

Reply:
Since the ONLY thing in `/usr/games` is xsnow, I updated with a `do_install` to just install things in sane places. I don't quite get the post_patch menthod posted above.

Reply:
I feel like it might make more sense for the cross-build `vsed` to be ran in `pre_build`.